lib/merch_calendar/date_calculator.rb

```diff
@@ -49,6 +51,17 @@ def end_of_month(year, merch_month)
       end
     end
 
+    def start_of_week(year, month, merch_week)
+      weeks = MerchCalendar.weeks_for_month(year, month)
+
+      week = weeks.find { |week| week.week == merch_week }
```

Comment:
Digging through the code, `MerchCalendar::MerchWeek.find(2017, 4, 1)` will give the same result.  So it'll clean this up here, though that code is doing something similar.